packages/lesswrong/server/dialogues/cron.ts

```diff
@@ -25,6 +26,30 @@ addCronJob({
   }
 });
 
+
+addCronJob({
+  name: 'notifyUsersOfTheirTurnInMatchForm',
+  interval: 'every 1 hour',
+  async job() {
+    const context = createAdminContext();
+    const checksYourTurn = await new DialogueChecksRepo().getMatchFormYourTurn()
+    checksYourTurn.forEach(check => {
+      void createNotification({
+        userId: check.userId,
+        notificationType: "yourTurnMatchForm",
+        documentType: null,
+        documentId: null,
```

Comment:
We have a `dialogueCheck` document type, which I'm pretty sure can be appropriately used here

Reply:
And then you don't need to pass `checkId` and `targetUserId` through `extraData`.